model/config.go

```diff
@@ -2835,6 +2840,15 @@ func (ss *SamlSettings) isValid() *AppError {
 		if !(*ss.CanonicalAlgorithm == SAML_SETTINGS_CANONICAL_ALGORITHM_C14N || *ss.CanonicalAlgorithm == SAML_SETTINGS_CANONICAL_ALGORITHM_C14N11) {
 			return NewAppError("Config.IsValid", "model.config.is_valid.saml_canonical_algorithm.app_error", nil, "", http.StatusBadRequest)
 		}
+
+		if len(*ss.GuestAttribute) > 0 {
+			if !(strings.Contains(*ss.GuestAttribute, "=")) {
```

Comment:
We need to decide it, but we can decide it later, is not needed to decide it before the merge.

Reply:
Yes, we still have an open issue in the Enterprise PR for this feature. If that changes, I will need to change this as well.